client/src/templates/Challenges/classic/desktop-layout.tsx

```diff
@@ -77,18 +78,25 @@ const DesktopLayout = (props: DesktopLayoutProps): JSX.Element => {
     layoutState,
     preview,
     hasEditableBoundaries,
-    superBlock
+    superBlock,
+    usesMultifileEditor
   } = props;
 
   const challengeFile = getChallengeFile();
   const projectBasedChallenge = showUpcomingChanges && hasEditableBoundaries;
-  const isPreviewDisplayable = projectBasedChallenge
+  const displayPreview = projectBasedChallenge
     ? showPreview && hasPreview
     : hasPreview;
-  const isConsoleDisplayable = projectBasedChallenge ? showConsole : true;
+  const displayConsole = projectBasedChallenge ? showConsole : true;
   const { codePane, editorPane, instructionPane, previewPane, testsPane } =
     layoutState;
 
+  const displayEditorTabs =
```

Comment:
Why not just `usesMultifileEditor` here? In which case, you could just use that as the check below.

I gave it a quick test, you do need at least a check for `!hasEditableBoudaries` or the tabs show up twice. But, I'm wondering, why the editor tabs are ever needed on the desktop view - wouldn't they always be in the `ActionRow` when on desktop?

Reply:
Overabundance of caution, to be honest.  I wanted to be sure the tabs were definitely hidden until we removed the `SHOW_UPCOMING_CHANGES` flag.

I'm fine with totally removing them for the desktop, though.